### DISCUSSION TITLE
otbr-agent is getting restarted continuously and getting "connect session failed: No such file or directory" upon using ot-ctl

### DISCUSSION THREAD

**Discussion by Jayesh19101999** [openthread/openthread#8751] [Q&A] [answered]
pi@raspberrypi:~/ot-br-posix $ sudo systemctl status otbr-agent.service
● otbr-agent.service - OpenThread Border Router Agent
   Loaded: loaded (/lib/systemd/system/otbr-agent.service; enabled; vendor preset: enabled)
   Active: active (running) since Mon 2023-02-13 07:32:10 GMT; 1s ago
  Process: 18706 ExecStartPre=/usr/sbin/service mdns start (code=exited, status=0/SUCCESS)
 Main PID: 18728 (otbr-agent)
    Tasks: 1 (limit: 2200)
   Memory: 1.0M
   CGroup: /system.slice/otbr-agent.service
           └─18728 /usr/sbin/otbr-agent -I wpan0 -B eth0 spinel+hdlc+uart:///dev/ttyACM0 trel://eth0

Feb 13 07:32:08 raspberrypi systemd[1]: Starting OpenThread Border Router Agent...
Feb 13 07:32:10 raspberrypi systemd[1]: Started OpenThread Border Router Agent.
Feb 13 07:32:10 raspberrypi otbr-agent[18728]: [NOTE]-AGENT---: Backbone interface: eth0
Feb 13 07:32:10 raspberrypi otbr-agent[18728]: [NOTE]-AGENT---: Running 0.3.0-4f92c238e2
Feb 13 07:32:10 raspberrypi otbr-agent[18728]: [NOTE]-AGENT---: Thread version: 1.3.0
Feb 13 07:32:10 raspberrypi otbr-agent[18728]: [NOTE]-AGENT---: Thread interface: wpan0
Feb 13 07:32:10 raspberrypi otbr-agent[18728]: [NOTE]-AGENT---: Radio URL: spinel+hdlc+uart:///dev/ttyACM0
Feb 13 07:32:10 raspberrypi otbr-agent[18728]: [NOTE]-AGENT---: Radio URL: trel://eth0
Feb 13 07:32:10 raspberrypi otbr-agent[18728]: [NOTE]-ILS-----: Infra link selected: eth0
Feb 13 07:32:10 raspberrypi otbr-agent[18728]: [INFO]-NCP-----: OpenThread log level changed to 4
pi@raspberrypi:~/ot-br-posix $ sudo systemctl status otbr-agent.service
● otbr-agent.service - OpenThread Border Router Agent
   Loaded: loaded (/lib/systemd/system/otbr-agent.service; enabled; vendor preset: enabled)
   Active: activating (auto-restart) (Result: exit-code) since Mon 2023-02-13 07:33:50 GMT; 2s ago
  Process: 18951 ExecStartPre=/usr/sbin/service mdns start (code=exited, status=0/SUCCESS)
  Process: 18973 ExecStart=/usr/sbin/otbr-agent $OTBR_AGENT_OPTS (code=exited, status=6)
 Main PID: 18973 (code=exited, status=6)
pi@raspberrypi:~/ot-br-posix $ sudo ot-ctl state
connect session failed: No such file or directory

hi i am trying to check the status of otbr-agent but sometimes its running and sometimes its not can someone help me out
I am following these steps https://openthread.io/guides/border-router/build

**Comment by jwhui** [comment, marked as answer]
The following error:
```
connect session failed: No such file or directory
```
Typically indicates that `otbr-agent` is not running. In most cases, `otbr-agent` quits when there is any issue communicating with the RCP.

You can view more of the `otbr-agent` logs with:
```
cat /var/log/syslog | grep otbr-agent
```
